.github/workflows/ReadBlob_release.yml

```diff
@@ -0,0 +1,14 @@
+name: ReadBlob_release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    uses: FrendsPlatform/FrendsTasks/.github/workflows/release.yml@main
+    with:
+      workdir: Frends.AzureBlobStorage.ReadBlob
+      env_var_name_1: HiQ_AzureBlobStorage_ConnString
```

Comment:
You're missing the Access Key secret from this workflow. Release also runs tests, and since you need the access key in tests, you also need it in this workflow.

Reply:
Connection string added to Actions secrets.